### DISCUSSION TITLE
TypeError: undefined is not an object (evaluating 'window.location.href')

### DISCUSSION THREAD

**Discussion by HassanIyan** [supabase/supabase#3754] [Questions] [answered]
i am getting this error when using supabase with expo

TypeError: undefined is not an object (evaluating 'window.location.href')
at node_modules/react-native/Libraries/Core/ExceptionsManager.js:104:6 in reportException
at node_modules/react-native/Libraries/Core/ExceptionsManager.js:172:19 in handleException
at node_modules/react-native/Libraries/Core/setUpErrorHandling.js:24:6 in handleError
at node_modules/@react-native/polyfills/error-guard.js:49:36 in ErrorUtils.reportFatalError
at node_modules/metro-runtime/src/polyfills/require.js:204:6 in guardedLoadModule
at http://192.168.100.31:19000/node_modules/expo/AppEntry.bundle?platform=android&dev=true&hot=false&minify=false:122154:3 in global code

Please help

**Comment by GaryAustin1** [comment, marked as answer]
See here: https://github.com/supabase/supabase-js/issues/295

**Reply by HassanIyan** [reply]
thank you very much... now the app works...
god bless you

**Reply by GaryAustin1** [reply]
This just came up again with a new post.  Can you mark this as answered so maybe others might catch it?
